Marlin/temperature.cpp

```diff
@@ -1809,13 +1812,41 @@ void Temperature::isr() {
       lcd_buttons_update();
       temp_state = MeasureTemp_3;
       break;
-    case MeasureTemp_3:
-      #if HAS_TEMP_3
-        raw_temp_value[3] += ADC;
-      #endif
-      temp_state = Prepare_FILWIDTH;
-      break;
```

Comment:
Here I suggest keeping `MeasureTemp3` mostly as it was, but just override the new state it sets:
```cpp
temp_state =
  #if ENABLED(E_ADJUST_MANUALLY)
    Prepare_e_adjust_reading
  #else
    Prepare_FILWIDTH
  #endif
;
```